Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:jammy
+
+RUN apt-get update
+RUN apt-get install git python3 pip -y
+
+
+# Build OpenOrb
+# The following are build requirements for oorb
+RUN apt-get install gfortran curl liblapack-dev python3-numpy wget -y
+RUN git clone https://github.com/oorb/oorb.git /oorb/
+WORKDIR /oorb/
+RUN ./configure gfortran opt  --with-pyoorb --with-python=python3 --with-f2py=/usr/bin/f2py3 --prefix=/usr/local
+RUN make
+RUN make ephem
+RUN cd /oorb/data && ./getBC430
+RUN cd /oorb/data && ./updateOBSCODE
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/oorb/python/"
+ENV PYTHONPATH="$PYTHONPATH:/oorb/python/"
+ENV OORB_DATA="/oorb/data"
+ENV OORB_CONF="/oorb/main/oorb.conf"
+RUN wget https://www.minorplanetcenter.net/iau/MPCORB/MPCORB.DAT
+RUN /oorb/bin/oorb --task=mpcorb --mpcorb=MPCORB.DAT > /oorb/data/MPCORB.orb
```

Comment:
Is this line to test if the oorb installation worked? The MPCORB is a fairly large file to download if we don't think we need it.

Reply:
I think something was failing without it (might have been a test of some kind?) I'll investigate.

Reply:
I removed it and I believe it's working, still.